core/arch/arm/pta/pta_cyres.c

```diff
@@ -313,6 +313,28 @@ static TEE_Result get_seal_key(struct cyres_pta_sess_ctx *ctx,
 
 	return res;
 }
+static TEE_Result handle_get_private_key_size(
+		struct cyres_pta_sess_ctx *ctx,
+		uint32_t param_types,
+		TEE_Param params[TEE_NUM_PARAMS])
+{
+	TEE_Result res;
+	size_t size;
```

Comment:
Should not this be initialized to 0?

Reply:
It doesn't need to be initialized since it's an out parameter. I had to initialize it in one spot because of a compiler warning.

Reply:
Ok, see it now. GitHub did not want to find the implementation for me earlier.